src/components/headerNav.js

```diff
@@ -0,0 +1,99 @@
+
+import React,{ Component } from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { push } from 'react-router-redux'
+
+import { Route, Link } from 'react-router-dom'
+
+import AppBar from 'material-ui/AppBar';
+import IconButton from 'material-ui/IconButton';
+import IconMenu from 'material-ui/IconMenu';
+import MenuItem from 'material-ui/MenuItem';
+import FlatButton from 'material-ui/FlatButton';
+import Toggle from 'material-ui/Toggle';
+import MoreVertIcon from 'material-ui/svg-icons/navigation/more-vert';
+import NavigationClose from 'material-ui/svg-icons/navigation/close';
+
+import Avatar from 'material-ui/Avatar';
+import RaisedButton from 'material-ui/RaisedButton';
+import '../css/index.css'
+
+class headerNav extends Component{
```

Comment:
Inconsistent with naming convention used for other classes.

Reply:
This has been fixed

Reply:
This has been fixed please approve